src/utils/changelog.ts

```diff
@@ -250,9 +253,16 @@ export async function generateChangesetFromGit(
     const hash = gitCommit.hash;
 
     const githubCommit = githubCommits[hash];
+    if (
+      githubCommit?.prBody &&
+      githubCommit.prBody.includes(SKIP_CHANGELOG_MAGIC_WORD)
```

Comment:
Why would this not work here?

```suggestion
      githubCommit?.prBody?.includes(SKIP_CHANGELOG_MAGIC_WORD)
```

Reply:
It does, it is just needed as we already establish the fact that both `githubCommit` and `githubCommit.prBody` exists. See [optional chaining operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) if you are curious 🙂

Reply:
Sorry ... maybe wasn't clear, why have the initial `githubCommit?.prBody &&` if we can just use optional chaining?

Reply:
Oh, I guess I can do that 😅

Reply:
#298